src/LanguageFunctions.ts

```diff
@@ -24,20 +26,39 @@ export async function FindNextUnTranslatedText(searchCurrentDocument: boolean):
         filesToSearch = (await WorkspaceFunctions.GetLangXlfFiles(vscode.window.activeTextEditor ? vscode.window.activeTextEditor.document.uri : undefined));
     }
 
-
     for (let i = 0; i < filesToSearch.length; i++) {
         const xlfUri = filesToSearch[i];
-        let searchResult = await DocumentFunctions.searchTextFile(xlfUri, startOffset, GetNotTranslatedToken());
-        let searchedForReviewtoken = false;
-        if (!searchResult.foundNode) {
-            searchResult = await DocumentFunctions.searchTextFile(xlfUri, startOffset, GetReviewToken());
-            searchedForReviewtoken = true;
+        if (useExternalTranslationTool) {
+            return (await FindNextUntranslatedByState(xlfUri, startOffset));
```

Comment:
Yes, that kind of solves it. But I realized that the current behavior isn't optimal either. When the search is performed with `searchCurrentDocument == true` it gets stuck on the first file since it's always sorted first in the array.

In my test case *en-US.xlf is the first file. So if start searching from *se-SV.xlf it will wrap the search until I reach *en-US.xlf where it will start to search only that file because it's always first in the array. I fixed it by moving the first element of filesToSearch if it has the same path as the current document. It's not the prettiest thing but it will suffice. 

I also noticed that due to the order in which the token search is performed it skips the ReviewToken because it's always searching for the NotTranslatedToken first. Since it's already present in the released version and I suggest we make it a separate issue so we can complete this PR.